payments.md

```diff
@@ -0,0 +1,161 @@
+## BusinessPayment
+
+## Payment collection [/payments]
+payments endpoint allows you to make a Payment anonymously or by logging in for Britishgas business customers. Responses will be returned in JSON format only.
+
+### BusinessPayment
```

Comment:
How can i use this resource and its attributes for making payment with the following scenarios. 
a) Payment with a saved card.
b) Payment using a new card.
c) Payment using anonymous customer.
d) Payment using a logged in customer.

This resource doesn't adhere to all the above scenarios. Please rework.